public/manifest.webmanifest

```diff
@@ -3,12 +3,12 @@
   "name": "COVID-19 Dashboard",
   "icons": [
     {
-      "src": "/android-chrome-192x192.png",
+      "src": "android-chrome-192x192.png",
```

Comment:
Parcel didn't like the absolute URL, because it tries to actually resolve file references to make sure they are valid. (This is how I found that the other icon was named wrong!)

Using relative URLs should not be an issue for the `/get-involved` page. Icon URLs specified in the web manifest file are resolved relative to the web manifest file, not the HTML page.
https://stackoverflow.com/a/59275500/782045